dev-libs/xerces_c/patches/xerces_c-3.2.1.patchset

```diff
@@ -0,0 +1,24 @@
+From 861fd54b041d5d2952435d93387209ccded76576 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sun, 24 Jun 2018 18:42:16 +0200
+Subject: define _BSD_SOURCE
+
+
+diff --git a/src/xercesc/util/XMLDateTime.cpp b/src/xercesc/util/XMLDateTime.cpp
+index 07dac1a..3345481 100644
+--- a/src/xercesc/util/XMLDateTime.cpp
++++ b/src/xercesc/util/XMLDateTime.cpp
+@@ -26,6 +26,10 @@
+ #    include <config.h>
+ #endif
+ 
++#if defined(__HAIKU__)
++#define _BSD_SOURCE
```

Comment:
If you add `-D_BSD_SOURCE` into `CPPFLAGS` instead, this wouldn't be necessary :)